### ISSUE TITLE
IInteractiveWindow.UIThreadOnly.SubmitAsync should allow to send empty lines

### ISSUE THREAD

**Issue by AlexanderSher** [dotnet/roslyn#8569]
```csharp
            private async Task SubmitAsync()
            {
                try
                {
                    RequiresLanguageBuffer();

                    // TODO: queue submission
                    // Ensure that the REPL doesn't try to execute if it is already
                    // executing.  If this invariant can no longer be maintained more of
                    // the code in this method will need to be bullet-proofed
                    if (State == State.ExecutingInput)
                    {
                        return;
                    }

                    // get command to save to history before calling FinishCurrentSubmissionInput
                    // as it adds newline at the end
                    var historySpan = CurrentLanguageBuffer.CurrentSnapshot.GetExtent();
                    FinishCurrentSubmissionInput();

                    _history.UncommittedInput = null;

                    var snapshotSpan = CurrentLanguageBuffer.CurrentSnapshot.GetExtent();
                    var trimmedSpan = snapshotSpan.TrimEnd();

                    if (trimmedSpan.Length == 0)
                    {
                        // TODO: reuse the current language buffer
                        PrepareForInput();
                        return;
                    }
```
It may be required to send empty/whitespace only line to IInteractiveEvaluator. Would be great to have a flag for that.

**Comment by AlexanderSher** [comment]
https://github.com/dotnet/roslyn/blob/8eb34a73f658fd9017f53763152570d59cbb0bde/src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs#L3286

**Comment by ManishJayaswal** [comment]
This I believe is fixed by #9894
